### PR TITLE
changes to comment and cleaning unwanted

### PR DESCRIPTION
**Pull request by petermugabo250**
hello check me

### CODE REVIEW

File: src/controller/blogcontroller.js [hunk 1]
```diff
@@ -94,31 +96,13 @@ export const allcomment = async (req, res) => {
...
 export const allBlogs = async (req, res) => {
   try {
-    const gettaallinfo = await blogmode.find();
+    const gettaallinfo = await blogmode.find().populate({path:'comment', select: 'CommentMessage user username userphoto'});
     return res.status(200).json({
       statusbar: "You Made It",
       message: "All Blogs Are here:",
       data: gettaallinfo,
     });
-// use agrgregate to retrieve Data
```
Comment (muhozajohn) [inline src/controller/blogcontroller.js:103]: Remove this Comment

**Review by muhozajohn — APPROVED** [review]

**Review by muhozajohn — APPROVED** [review]

File: src/controller/blogcontroller.js [hunk 2]
```diff
@@ -36,30 +36,32 @@ export const createComment = async (req, res) => {
   try {
     const { id } = req.params;
     const { usertable } = req;
-    const { CommentMessage,blogId } = req.body;
-    const blog = await blogmode.findById(id);
-    if (!blog) {
+    const { CommentMessage} = req.body;
+    const post = await blogmode.findById(id);
```
Comment (mansurissa) [inline src/controller/blogcontroller.js]: REname this to blog blogModel (Use camelCase please)

File: src/controller/blogcontroller.js [hunk 3]
```diff
@@ -36,30 +36,32 @@ export const createComment = async (req, res) => {
...
+    if (!post) {
       return res.status(404).json({
         status: "404",
         message: "Blog Not found",
       });
     }
-    const comm = await usercomments.create({
+    const comments = await usercomments.create({
       CommentMessage,
       user: usertable._id,
```
Comment (mansurissa) [inline src/controller/blogcontroller.js]: use camelCase so userTable

File: src/controller/blogcontroller.js [hunk 4]
```diff
@@ -94,31 +96,13 @@ export const allcomment = async (req, res) => {
 
 export const allBlogs = async (req, res) => {
```
Comment (mansurissa) [inline src/controller/blogcontroller.js]: rename this to something like getAllPosts

File: src/controller/blogcontroller.js [hunk 5]
```diff
@@ -94,31 +96,13 @@ export const allcomment = async (req, res) => {
...
   try {
-    const gettaallinfo = await blogmode.find();
+    const gettaallinfo = await blogmode.find().populate({path:'comment', select: 'CommentMessage user username userphoto'});
     return res.status(200).json({
       statusbar: "You Made It",
       message: "All Blogs Are here:",
       data: gettaallinfo,
```
Comment (mansurissa) [inline src/controller/blogcontroller.js]: Use an understandable variable names

File: src/controller/blogcontroller.js [hunk 6]
```diff
@@ -36,30 +36,32 @@ export const createComment = async (req, res) => {
...
         message: "Blog Not found",
       });
     }
-    const comm = await usercomments.create({
+    const comments = await usercomments.create({
       CommentMessage,
       user: usertable._id,
-      PostId:blog._id,
+      PostId:post._id,
+      username: usertable.lastname,
```
Comment (mansurissa) [inline src/controller/blogcontroller.js]: It seems like `usertable` is the loggedn in user, go back in your authentication middleware and rename it to something like `loggedInUser` or `authenticatedUser`.

**Review by mansurissa — CHANGES_REQUESTED** [review]
Go back in your whole project and rename`blog` to `post`, user camelCase for all of your varaible names and file names, and make sure your response has : status, message, data/error and work on the rest of the comments I left while reviewing. Happy hacking

File: src/controller/blogcontroller.js [hunk 7]
```diff
@@ -1,31 +1,32 @@
...
         PostContent,
-      creator: req.usertable.lastname,
-      creatorprofile: req.usertable.profile,
-      PostedDate: req.usertable.PostedDate,
+      creator: req.authenticatedUser.lastname,
+      creatorprofile: req.authenticatedUser.profile,
+      PostedDate: req.authenticatedUser.PostedDate,
     });
     return res.status(200).json({
-      message: "Your Blog have been created",
```
Comment (muhozajohn) [inline src/controller/blogcontroller.js:22]: add status code

File: src/controller/blogcontroller.js [hunk 8]
```diff
@@ -1,31 +1,32 @@
...
-      data: Blog,
+      message: "Your Post have been created",
+      data: Post,
     });
   } catch (error) {
     return res.status(500).json({
-      statusbar: "Failed",
-      message: "Failied To Create Blog",
+      statusbar: "500",
```
Comment (muhozajohn) [inline src/controller/blogcontroller.js]: use status not statusbar

File: src/controller/blogcontroller.js [hunk 9]
```diff
@@ -35,51 +36,54 @@ export const createBlog = async (req, res) => {
...
+    const gettAllcomment = await Comments
       .find({PostId:id,}).populate("user","firstname lastname ");
     return res.status(200).json({
       status: "200",
       message: "All comments Are here:",
-      data: gettallcomment,
+      data: gettAllcomment,
     });
   } catch (error) {
     console.log(error.message)
```
Comment (muhozajohn) [inline src/controller/blogcontroller.js]: add return function on catch that will cath errors and stutas code

File: src/controller/blogcontroller.js [hunk 10]
```diff
@@ -90,47 +94,29 @@ export const allcomment = async (req, res) => {
   }
 };
 
-//getting all data
+//getting allPosts
```
Comment (muhozajohn) [inline src/controller/blogcontroller.js]: Remove this Comment

File: src/controller/blogcontroller.js [hunk 11]
```diff
@@ -152,44 +138,46 @@ export const blogById = async (req, res) => {
     });
   }
 };
-// Deleting Blog By Blog Id
+// Deleting Post By Id
```
Comment (muhozajohn) [inline src/controller/blogcontroller.js]: Remove this comment

File: src/controller/userControllers.js [hunk 12]
```diff
@@ -206,7 +205,7 @@ export const GetUsers = async(req, res) =>{
   catch (error){
     
     return res.status(500).json({
-      statusbar: "Sorry Something Went Wrong",
+      statusbar: "500",
```
Comment (muhozajohn) [inline src/controller/userControllers.js:206]: change to status

**Review by muhozajohn — CHANGES_REQUESTED** [review]

**Review by muhozajohn — APPROVED** [review]

**Pull request merged**